app/views/layouts/application.html.erb

```diff
@@ -38,6 +38,10 @@
         </div>
       </nav>
 
+      <% flash.each do |name, msg| %>
```

Comment:
How did this code get into this branch? Make sure you *always branch from `master`*. Do not branch from a feature branch -- you cannot guarantee that branches will be merged in a specific order (or at all) so they should not depend on other branches.